.github/workflows/reusable_upload_web.yml

```diff
@@ -100,21 +96,37 @@ jobs:
           parent: false
 
       - name: "Upload web-viewer (prerelease)"
-        if: inputs.MARK_PRERELEASE_FOR_MAINLINE
+        if: github.ref == 'refs/heads/master'
         uses: google-github-actions/upload-cloud-storage@v1
         with:
           path: "web_viewer"
           destination: "rerun-web-viewer/prerelease"
           parent: false
 
       - name: "Upload RRD (prerelease)"
-        if: inputs.MARK_PRERELEASE_FOR_MAINLINE
+        if: github.ref == 'refs/heads/master'
         uses: google-github-actions/upload-cloud-storage@v1
         with:
           path: "rrd"
           destination: "rerun-example-rrd/prerelease"
           parent: false
 
+      - name: "Upload web-viewer (nightly)"
+        if: github.ref == 'refs/heads/master'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "web_viewer"
+          destination: "rerun-web-viewer/version/nightly"
+          parent: false
+
+      - name: "Upload RRD (nightly)"
+        if: github.ref == 'refs/heads/master'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "rrd"
+          destination: "rerun-example-rrd/version/nightly"
+          parent: false
```

Comment:
It is not run every night though, but on every merge into `main`. Maybe this should be called `trunk` instead?

Or do you envision this will be chnaged to nightly at some point, and so `nightly` is more of a promise of "at least once per day"?

Reply:
Yes, we want to build all the examples, ideally once per day, but we're not there yet. `version/nightly` will eventually become that, and I figured there's no point in temporarily calling it something else.

Reply:
Sounds good, but reading this code doesn't convey that - please add a comment!